build-system-abstraction.rst

```diff
@@ -0,0 +1,360 @@
+:PEP: XX
+:Title: Build system abstraction for pip/conda etc
+:Version: $Revision$
+:Last-Modified: $Date$
+:Author: Robert Collins <rbtcollins@hp.com>,
+         Nathaniel Smith <njs@pobox.com>
+:Discussions-To: distutils-sig <distutils-sig@python.org>
+:Status: Draft
+:Type: Standards Track
+:Content-Type: text/x-rst
+:Created: 26-Oct-2015
+:Post-History: XX
+:Requires: PEP-426
+
+
+Abstract
+========
+
+This PEP specifies a programmatic interface for pip [#pip]_ and other
+distribution or installation tools to use when working with Python
+source trees (both the developer tree - e.g. the git tree - and source
+distributions).
+
+The programmatic interface allows decoupling of pip from its current
+hard dependency on setuptools [#setuptools]_ able for two
+key reasons:
+
+1. It enables new boutique build systems that may be much easier to use
```

Comment:
"new boutique" doesn't sound right. "boutique" says to me "small/niche/specialized", while I'd hope to see something that becomes widely used/applicable fast. And there are also existing build systems. Maybe rephrase?